redfish-core/lib/service_root.hpp

```diff
@@ -123,6 +123,34 @@ inline void
     asyncResp->res.jsonValue["Oem"]["IBM"]["DateTimeLocalOffset"] =
         redfishDateTimeOffset.second;
 
+    // Report if ACFWindowActive
+    crow::connections::systemBus->async_method_call(
+        [asyncResp](const boost::system::error_code ec,
+                    const std::variant<bool>& retVal) {
+            if (ec)
+            {
+                BMCWEB_LOG_ERROR
+                    << "Failed to read ACFWindowActive property";
+            }
+            else
+            {
+                const bool* isACFWindowActive = std::get_if<bool>(&retVal);
+                if (isACFWindowActive == nullptr)
+                {
+                    BMCWEB_LOG_ERROR << "nullptr for ACFWindowActive";
+                    messages::internalError(asyncResp->res);
```

Comment:
more typical in bmcweb is 
            if (ec)
            {
                BMCWEB_LOG_ERROR
                    << "Failed to read ACFWindowActive property";
                    
                    return;
            }
            
                const bool* isACFWindowActive = std::get_if<bool>(&retVal);
                if (isACFWindowActive == nullptr)
                {
                    BMCWEB_LOG_ERROR << "nullptr for ACFWindowActive";
                    messages::internalError(asyncResp->res);
                    return;
                 }

Reply:
Thanks.   I'll break out handleACFWindowActive as a separate function so I can structure the code like this.